src/test/java/bio/terra/workspace/service/workspace/WorkspaceServiceTest.java

```diff
@@ -104,7 +104,7 @@ void testGetExistingWorkspace() {
   void testGetForbiddenMissingWorkspace() {
     doThrow(new SamUnauthorizedException("forbid!"))
         .when(mockSamService)
-        .workspaceAuthzOnly(any(), any(), any());
+        .checkAuthz(any(), any(), any(), any());
```

Comment:
nit: You could put these tests in an `@Nested` and have a second `@BeforeEach` that sets up the mock.

Reply:
I kind of like having the non-standard mock setup right next to the test, even though this is duplicated in a few places